frontend/src/app/main/ui/measurements.cljs

```diff
@@ -261,3 +270,188 @@
+                               (when @resizing?
+                                 (reset! resizing? false)
+                                 (st/emit! (dwm/apply-modifiers)
+                                           (dwsl/update-layout [frame-id] {:layout-padding-type (if (and (= (:p1 padding) (:p3 padding)) (= (:p2 padding) (:p4 padding))) :simple :multiple) }))
+                                 (on-lost-pointer-capture e)))]
+
+    (mf/use-effect
+     (mf/deps frame-id padding-num delta)
+     (fn []
+       (when @resizing?
+         (let [new-value (int (max (+ @original-value (/ delta zoom)) 0))
+               modifiers (dwm/create-modif-tree [frame-id] (ctm/change-property (ctm/empty) :layout-padding (assoc padding padding-num new-value)))]
+           (st/emit! (dwm/set-modifiers modifiers))))))
+    [:*
+     [:rect.padding-rect {:x (:x rect-data)
+                          :y (:y rect-data)
+                          :width (:width rect-data)
+                          :height (:height rect-data)
+                          :on-mouse-enter (:mouse-enter rect-data)
+                          :on-mouse-leave (:mouse-leave rect-data)
+                          :on-pointer-down pointer-down
+                          :on-lost-pointer-capture lost-pointer-capture
+                          :on-mouse-move on-mouse-move
+                          :style {:fill (if (or hover? selected?) distance-color "none")
+                                  :cursor (when (or hover? selected?)
+                                            (if (= (:resize-axis rect-data) :x) (cur/resize-ew 0) (cur/resize-ew 90)))
+                                  :opacity (if selected? 0.5 0.25)}}]
+     (when (or hover? selected?)
+       [:& padding-display-pill pill-data])]))
+
+(mf/defc padding-rects [{:keys [frame zoom]}]
+  (let [frame-id (:id frame)
+        paddings-selected (mf/deref refs/workspace-paddings-selected)
+        hover    (mf/use-state nil)
+        padding (:layout-padding frame)
+        [width height x1 x2 y1 y2] ((juxt :width :height :x1 :x2 :y1 :y2) (:selrect frame))
+
+        pill-width (/ padding-pill-width zoom)
+        pill-height (/ padding-pill-height zoom)
+        pill-separation (/ pill-width 2)
+        pill-data {:height pill-height
+                   :width pill-width
+                   :font-size (/ font-size zoom)}
+        mouse-enter #(reset! hover %)
+        mouse-leave #(reset! hover nil)
+        negate {:p1 (if (:flip-y frame) true false)
+                :p2 (if (:flip-x frame) true false)
+                :p3 (if (:flip-y frame) true false)
+                :p4 (if (:flip-x frame) true false)}
+
+        negate (cond-> negate
+                       (= :fix (:layout-item-h-sizing frame)) (assoc :p2 (not (:p2 negate)))
+                       (= :fix (:layout-item-v-sizing frame)) (assoc :p3 (not (:p3 negate))))]
+
+    [:g.paddings {:pointer-events "visible"}
+     [:& padding-display {:frame-id frame-id
```

Comment:
Maybe you could create a `for` loop for these?

Reply:
Done